requirements.txt

```diff
@@ -4,7 +4,6 @@ coloredlogs
 marshmallow
 ruamel.yaml
 SQLAlchemy
-python-dateutil
```

Comment:
What exactly were the issues with this? We use it in more repo's than HK, does it also cause issues there?

Reply:
It is deprecated code. Might not cause issues now but since it is not maintained it can cause silent issues in the future